packages/hooks/src/useTransferMessageBetweenWalletAndTradingApp.ts

```diff
@@ -0,0 +1,67 @@
+import useExchangeRate from './useExchangeRate';
+import useWalletTransfer from './useWalletTransfer';
+
+type TTransferMessageReturnType = {
+    code: 'WalletToTradingAppDailyLimit' | 'DemoWalletToTradingAppDailyLimit';
+    cta?: string;
+    is_first_transfer: boolean;
+    limit?: number;
+    currency?: string;
+    type: 'error' | 'info' | 'success';
+};
+
```

Comment:
Need suggestions for caching `checkIsTradingAccount` and `tradingAccountMapper` the functions here.
cc: @heorhi-deriv @farzin-deriv 🙏